src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped");
```

Comment:
Similarly, the stopWork method's message should also be more informative. It should clearly indicate that the Excavator has stopped working.